ec-europa/eubfr-data-lake
# feat(projects): save results in DynamoDB - EUBFR-32

Test scenario:

1. upload a CSV into your main bucket
2. check if the `[original_filename].ndjson` file has been created in your harmonized bucket
3. check if the records have been saved in your `[stage]-projects` table
4. delete the original CSV
5. make sure that the `[original_filename].ndjson` file has been deleted as well
6. check if the records have been deleted from the table

You could also try an "update" scenario. Upload a CSV, then edit the CSV (remove one line) and upload it again. The record should have been deleted

![save](https://user-images.githubusercontent.com/3524389/31176872-a477d9ea-a914-11e7-88f1-c886ee3661fe.jpg)
